### ISSUE TITLE
Nextcloud 22 update issue

### ISSUE THREAD

**Issue by kmsgli** [nextcloud/server#28099]
### Steps to reproduce
1. Run pacman -Syu (system update for arch linux)
2. open web address to nextcloud server
3. Run update in web browser

### Expected behaviour
Login to nextcloud like normal

### Actual behaviour
Error shows up;

```
Preparing update

Set log level to debug

Turned on maintenance mode

Repair step: Repair MySQL collation

Repair info: All tables already have the correct collation -> nothing to do

Repair step: Repair SQLite autoincrement

Repair step: Copy data from accounts table when migrating from ownCloud

Repair step: Drop account terms table when migrating from ownCloud

Updating database schema

Exception: Database error when running migration latest for app core
```

### Server configuration

**Operating system:**
Arch Linux
**Web server:**
Apache
**Database:**
mysql
**PHP version:**
php8
**Nextcloud version:** (see Nextcloud admin page)
22
**Updated from an older Nextcloud/ownCloud or fresh install:**
update from 21
**Where did you install Nextcloud from:**
arch linux repos
**Signing status:**
<details>
<summary>Signing status</summary>

```
Login as admin user into your Nextcloud and access 
http://example.com/index.php/settings/integrity/failed 
paste the results here.
```
</details>

**List of activated apps:**
<details>
<summary>App list</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```
</details>

**Nextcloud configuration:**
<details>
<summary>Config report</summary>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or 

Insert your config.php content here. 
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```
</details>

**Are you using external storage, if yes which one:** local/smb/sftp/...

**Are you using encryption:** yes/no

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

#### LDAP configuration (delete this part if not used)
<details>
<summary>LDAP config</summary>

```
With access to your command line run e.g.:
sudo -u www-data php occ ldap:show-config
from within your Nextcloud installation folder

Without access to your command line download the data/owncloud.db to your local
computer or access your SQL server remotely and run the select query:
SELECT * FROM `oc_appconfig` WHERE `appid` = 'user_ldap';


Eventually replace sensitive data as the name/IP-address of your LDAP server or groups.
```
</details>

### Client configuration
**Browser:**

**Operating system:**

#### Web server error log
<details>
<summary>Web server error log</summary>

```
Insert your webserver log here
```
</details>

#### Nextcloud log (data/nextcloud.log)
<details>
<summary>Nextcloud log</summary>

```
Insert your Nextcloud log here
```
</details>

**Comment by goekce** [comment]
https://help.nextcloud.com/t/update-to-next-cloud-21-0-2-has-get-an-error/117028/7 could help

**Comment by kmsgli** [comment]
It seems it has something to do with updating database schema because that is the last thing it does. Not sure what you linked is related the error seems a little different.

**Comment by kmsgli** [comment]
So i figured it would be faster to re install nextcloud and it seems something is wrong still. Delete the database delete the config file re install gives me this error

`Error while trying to initialise the database: An exception occurred while executing a query: SQLSTATE[HY000]: General error: 4047 InnoDB refuses to write tables with ROW_FORMAT=COMPRESSED or KEY_BLOCK_SIZE. `

**Comment by kmsgli** [comment]
I have determined the issue not sure exactly why but hopefully someone can chime in with a bit more knowledge. I downgraded mariaDB from 10.6.3-1 => 10.5.9-1 (this was the last version in my pacman cache ) there is a version or two between those but i figured i would go with what was last installed and that solved my problem. Of course I already reset my server which was a waste of time since that was not the issue.

It seems the issue is with nextcloud  and the most current version of MariaDB. I initially thought it was the upgrade to nextcloud 22 but when i downgraded to nextcloud 21 it did not fix my issue. It was timing that nextcloud and mariaDB happened to both get updates at the same time that threw me off. For now I have set MariaDB not to update but this is a short term solution wondering if anyone else running Arch has come across this problem.

Thanks.
